netmiko/vyos/vyos_ssh.py

```diff
@@ -132,9 +132,12 @@ def send_config_set(
 
     def save_config(
         self,
-        cmd: str = "copy running-config startup-config",
+        cmd: str = "save",
```

Comment:
Did this change in vyos at some point? I would have thought the original contributor tested it so I am wondering why it used to work and now doesn't (though maybe it was never tested, I would have to look through some history to figure that out).